### ISSUE TITLE
Private DNS

### ISSUE THREAD

**Issue by kmarinov7** [awsdocs/amazon-vpc-user-guide#20]
In this page - https://docs.aws.amazon.com/vpc/latest/userguide/endpoint-service.html under Endpoint Service Limitations is explicitly mentioned:

"Service consumers must use the endpoint-specific DNS hostnames to access the endpoint service. Private DNS is not supported. For more information, see Accessing a Service Through an Interface Endpoint."

However when I go to Accessing a Service Through an Interface Endpoint (https://docs.aws.amazon.com/vpc/latest/userguide/vpce-interface.html#access-service-though-endpoint) I can see the following:

"If you have enabled private DNS for the endpoint (a private hosted zone), the default DNS hostname for the AWS service for the Region; for example, ec2.us-east-1.amazonaws.com."

---

In other words - we CAN or we CANNOT use private DNS for VPC service endpoint?

Thank you in advance

**Comment by katerini** [comment]
Hi, thanks for raising this issue. To answer your question: if you're accessing an AWS service or an AWS Marketplace partner service through an interface endpoint, you can enable and use private DNS. 
However, if you're accessing services hosted by other AWS customers (endpoint services), private DNS is not supported.

I see how this is confusing in the docs - the limitation is stated in the endpoint services section, but the link points to a different section of the docs that includes a discussion of using private DNS when connecting to AWS services. I'll try make this distinction clearer in the docs.
